src/modules/oAuth/oAuth.controller.ts

```diff
@@ -23,14 +25,19 @@ export class OAppController {
 
   @Get('google/callback')
   @UseGuards(GoogleOAuthGuard)
-  async googleAuthRedirect(@GetUser() user, @Response() res) {
+  async googleAuthRedirect(@GetUser() user, @Res() res: Response) {
     if (!user) {
       throw new UnauthorizedException('User is not authenticated');
     }
+
     const tokens = await this.appService.googleLogin(user);
-    res.cookie('accessToken', tokens.accessToken, { httpOnly: true });
-    res.cookie('refreshToken', tokens.refreshToken, { httpOnly: true });
-    // we can redirect to the frontend app when the user is authenticated
-    res.redirect(this.configService.get<string>('GOOGLE_CLIENT_REDIRECT_URL'));
+    res.cookie('refreshToken', tokens.refreshToken, {
+      httpOnly: true,
+      secure: this.configService.get('NODE_ENV') === 'production',
```

Comment:
Maybe make a function to check node environment? Coz it's frequently used.

Reply:
done